gadget/gadget.go

```diff
@@ -531,6 +534,89 @@ type VolumeUpdate struct {
 	Preserve []string       `yaml:"preserve" json:"preserve"`
 }
 
+// VolumeAssignment is an optional set of volume-to-disk assignments
+// that can be specified. This is a nice way of reusing the same gadget
+// for multiple devices, or a way to map multiple volumes to the same disk
+// for cases like eMMC. Each assignment in this structure refers to a volume
+// and a device path (/dev/disk/** for now).
+type VolumeAssignment struct {
+	Name        string                       `yaml:"name"`
```

Comment:
I know the spec uses 'name', but I feels slightly too generic. What if this was called `variant-name` ?

Reply:
The spec does mention the naming of `assignment-name`, maybe that instead? Or do you still feel better about `variant-name`?

Reply:
Renamed in 22fc49615178a3f914b9356c6969d87d3dba0649